src/main/java/frc/robot/Ports.java

```diff
@@ -59,7 +59,17 @@ public static final class SwerveDrive {
         };
     }
 
-    public static class UI {
+    public static final class Elevator {
+        public static final int MAIN_MOTOR  = 0;
+        public static final int AUX_MOTOR = 0;
+        public static final int TOP_LIMIT_SWITCH= 0;
+        public static final int BOTTOM_LIMIT_SWITCH= 0;
```

Comment:
These should be wired directly to the motor controller.

Reply:
Not sure how

Reply:
Look for limit switch configurations in the documentation or the API reference.

Reply:
36b85ab1ea79db7c196561265572b3c460aa3bae

Reply:
1. 3b95a65c76d5f644ddf930f852b5e8aed001a2b7
2. a45968d1de163a2f4e58fcb17bcb6851ee5ddca4

Reply:
2. What I meant is why not to do like you do in the following code, where you create the object directly where you use it:
https://github.com/Galaxia5987/robot-2024/blob/a45968d1de163a2f4e58fcb17bcb6851ee5ddca4/src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java#L61-L63

Reply:
c1e7da150f9c515fcc65871fcb219da08c3e578a

Reply:
Nice!